src/core/config.go

```diff
@@ -119,6 +120,12 @@ func loadJira() (Jira, error) {
 	}
 	jira.URL = viper.GetString("jira.url")
 
+	if viper.IsSet("jira.closed.status") {
```

Comment:
always execute : `jira.ClosedStatus = viper.GetString("jira.closed.status")`

and add a viper.Default() before